tests/test_exacttarget_discover.py

```diff
@@ -1,101 +0,0 @@
-import datetime
```

Comment:
Instead of removing the discovery test, could you update it to meet current standards https://github.com/stitchdata/tap-tester/blob/master/reference/expectations/saas_taps.md#test_discoverypy

Reply:
Yes, it is added as part of PR #68.